core_backend/app/configs/llm_prompts.py

```diff
@@ -89,21 +90,33 @@ def get_prompt(cls) -> str:
         [{",".join(cls._member_names_)}]
         """
 
+    @classmethod
+    def get_supported_languages(cls) -> list[str]:
+        """
+        Returns a list of supported languages.
+        """
+        return [lang for lang in cls._member_names_ if lang != "UNKNOWN"]
 
-# ----  Translation bot
 
-TRANSLATE_INPUT = """
+# ----  Translation bot
+TRANSLATE_FAILED_MESSAGE = "ERROR: CAN'T TRANSLATE"
+TRANSLATE_INPUT = f"""
     You are a high-performing translation bot for low-resourced African languages.
-    You support a maternal and child health chatbot.
+    You support a question-answering chatbot.
+    If you are unable to translate the user's input,
+    respond with {TRANSLATE_FAILED_MESSAGE}
     Translate the user's input to English from """
 
 # ---- Paraphrase question
-
-PARAPHRASE_INPUT = """
+PARAPHRASE_FAILED_MESSAGE = "ERROR: CAN'T PARAPHRASE"
+PARAPHRASE_INPUT = f"""
     You are a high-performing paraphrase bot.
-    You support a maternal and child health chatbot.
-    The user has asked a health question in English, paraphrase it to focus on
-    the health question. Be succinct and do not include any unnecessary information."""
+    You support a question-answering service.
+    The user has asked a question in English, paraphrase it to focus on
+    the question. Be succinct and do not include any unnecessary information.
+    Ignore any redacted and offensive words.
```

Comment:
What does redacted mean in this context?

Reply:
If there was swearing in hindi and you translated that to english, gpt4 will redact that swear word when translating. it will show up as something like "Since you are a [REDACTED], can you answer my questions". We want paraphrasing to ignore those redacted words.

Reply:
Achha thanks :)